### ISSUE TITLE
Feature request: Allow user to paste the image from clipboard with a CLRL+V.

### ISSUE THREAD

**Issue by raindropsfromsky** [manisandro/gImageReader#448]
A typical scenario is that the user has taken a screenshot (which is there on the clipboard), and he wants to paste it in the gImageReader. There is no keyboard shortcut for this.

Please assign CTRL+V for this task.

**Comment by manisandro** [comment]
CTRL+V very much depends on the focused widget though, you could assign it just to the canvas/source tree, but then the user won't get the desired behaviour is say he had some control input focused. I'd rather keep just the function in the sources menu, you'll be using the mouse for most tasks at this stage anyway.

**Comment by raindropsfromsky** [comment]
Exactly!

I right-clicked on the main panel and expected a context menu, with the **Paste** option.
But it is missing.

That's the most natural to do (if not CTRL+V). In the beginning, I am looking at the main panel, which is the most dominating part in the screen; and so it has my focus anyway. So when I press CTRL+V, I do not expect any other output panel to catch that and respond.

**Comment by manisandro** [comment]
What is the "main panel"?

The option is here:

![image](https://user-images.githubusercontent.com/1298852/79129074-d8956780-7da4-11ea-87be-2b105102f0d2.png)

**Comment by raindropsfromsky** [comment]
I call the area in red rectangle as the main panel:
![image](https://user-images.githubusercontent.com/9047168/79132905-bc0b2680-7dc8-11ea-9760-19f0aa7c1f2e.png)

I right-clicked there, and expected a context menu to paste the clipboard content there.

**Comment by manisandro** [comment]
Well you kinda create a new file with the respective contents, so not completely out-of-place in my opinion.

Having it in the context menu of the canvas generally only would make sense if the canvas is empty, as it's not a interaction with any loaded image.

**Comment by raindropsfromsky** [comment]
Instead, why not place all buttons in a single pane? 
Like this:

Add folder       
Recent docs...
Take a screenshot
Paste
Acquire...  
(The **Acquire** sub-options pop up when you select it.)

No tabs. Much simpler...

And of course, please provide that **Paste** option as context menu option when we right-click "anywhere". If there is already an image in the main screen, gImageReader should ask whether to discard the older image, or save it before pasting, or cancel the paste option.

Thus we have three options:
1. Save the old figure and paste the new content
2. Discard the old figure and paste the new content
3. Cancel the paste operation (and continue with the old stuff)

**Comment by raindropsfromsky** [comment]
BTW I noticed on strange behavior: If I click on the **open folder** icon, it looks for a single image/pdf file. But the drop-down menu next to it does not have this option at all.

This is strange because normally the drop-down menu headers do not have any function at all. Secondly, the drop-down menu has all the options. One of its options is never separated and hidden in its heading.

So I have updated my suggestion to the following:

Add image/pdf       [Recent docs...]
Add folder
Take a screenshot
Paste
Acquire...
(The Acquire sub-options pop up when you select it.)
